rust/src/aes.rs

```diff
@@ -0,0 +1,126 @@
+use crypto_pals::num_blocks;
+use hex::decode;
+use hex::encode;
+use openssl::error::ErrorStack;
+use openssl::symm::{decrypt, encrypt, Cipher};
+use rand::Rng;
```

Comment:
```suggestion
use rand::{Rng, RngCore};
```

Reply:
Currently doesn't compile without this, You need to `RngCore` trait for `next_u64()`